torch/utils/data/datapipes/utils/decoder.py

```diff
@@ -1,13 +1,15 @@
 # This file takes partial of the implementation from NVIDIA's webdataset at here:
 # https://github.com/tmbdev/webdataset/blob/master/webdataset/autodecode.py
 
+import io
+import os
 import pickle
 import re
-import os
+import tempfile
 
 import json
-import tempfile
-import io
+import numpy as np
```

Comment:
`numpy` is one of the requirements for pytorch, so it should be fine for direct import